.changes/next-release/bugfix-AWSSDKforJavav2-c277e61.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
```

Comment:
Nit: can we have separate entries for CRT HTTP client and Netty HTTP client? The description can probably be the same.